evap/static/ts/src/contact_modal.ts

```diff
@@ -10,7 +10,7 @@ export class ContactModalLogic {
     private readonly successMessageModal: bootstrap.Modal;
     private readonly actionButtonElement: HTMLButtonElement;
     private readonly messageTextElement: HTMLInputElement;
-    private readonly showButtonElement: HTMLElement;
+    private readonly showButtonElements: NodeListOf<HTMLElement>;
```

Comment:
If I recall correctly, node lists are a bit icky because they don't have some methods I expect (like map, filter), so I just always went with `Array` instead (see `quick-review-slider.ts`) - I am fine with both here though

Reply:
I just used the type that was returned by the API that I used, I think.

Reply:
But the class doesn't care about that. My point was that I personally prefer to have the rich / standard `Array` instead of the odd `NodeList` - even if we don't strictly need it (yet).